app/assets/stylesheets/application/member-card.scss

```diff
@@ -1,12 +1,36 @@
+.card-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: stretch;
+  gap: 20px;
+
+
+  @media(max-width: $screen-sm-min) {
+    display: block;
+  }
+}
+
+.card-wrapper {
+  width: 48%;
+
+  @media (max-width: $screen-sm-min) {
+    width: 100%;
+  }
+}
+
 .to-member-card {
   & {
     margin: 16px 0;
```

Comment:
I think it's better if we remove this margins as they are having a strange effect